lab3.md

```diff
@@ -438,6 +438,47 @@ You can now type in `faas-cli new --lang node10-express`.
 
 See also: [Function & Template Store](https://github.com/openfaas/store/)
 
+### Variable Substitution in YAML File (optional exercise)
```

Comment:
I like this a lot.

I wonder whether we can use it to show how to deploy a function ending in `-dev/-prod` in the function's name field?

The Docker image is a useful part to change, anything else for first-time users?